examples/falcon-7b-instruct/finetuned-model-custom-prompt.yaml

```
@@ -11,6 +11,8 @@ spec:
     name: k8s-instructions
   params:
     num_train_epochs: 1
+    # Save to checkpoint every 5 steps
+    save_steps: 5
```

Adding a little to the comment to help contextualize why you chose 5 for the given dataset:

```suggestion
    # Save to checkpoint every 5 steps for a dataset with ~70 steps total.
    save_steps: 5
```